[2022-11-16 01:32:30,127] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_world.goodbye_task 2022-02-21T00:00:00+00:00 [queued]>
[2022-11-16 01:32:33,479] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_world.goodbye_task 2022-02-21T00:00:00+00:00 [queued]>
[2022-11-16 01:32:33,704] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 01:32:33,772] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-11-16 01:32:33,878] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 01:32:35,466] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): goodbye_task> on 2022-02-21T00:00:00+00:00
[2022-11-16 01:32:35,681] {standard_task_runner.py:52} INFO - Started process 349 to run task
[2022-11-16 01:32:36,268] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_world', 'goodbye_task', '2022-02-21T00:00:00+00:00', '--job-id', '64', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/index.py', '--cfg-path', '/tmp/tmpy6jehskf', '--error-file', '/tmp/tmpdwinnue0']
[2022-11-16 01:32:36,332] {standard_task_runner.py:77} INFO - Job 64: Subtask goodbye_task
[2022-11-16 01:32:45,121] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.goodbye_task 2022-02-21T00:00:00+00:00 [running]> on host 001906ffd7c9
[2022-11-16 01:32:54,507] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=goodbye_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-21T00:00:00+00:00
[2022-11-16 01:32:54,727] {logging_mixin.py:104} INFO - bye all
[2022-11-16 01:32:54,737] {python.py:118} INFO - Done. Returned value was: bye
[2022-11-16 01:32:58,826] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=goodbye_task, execution_date=20220221T000000, start_date=20221116T013230, end_date=20221116T013258
[2022-11-16 01:33:04,657] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-11-16 01:33:05,786] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-11-16 01:33:07,194] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 349
[2022-11-16 01:33:07,845] {process_utils.py:66} INFO - Process psutil.Process(pid=349, status='terminated', exitcode=0, started='01:32:34') (349) terminated with exit code 0
